api/server.go

```diff
@@ -58,14 +59,14 @@ func (a *API) createLoadAgentHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	ltConfig := data.LoadTestConfig
-	if err := ltConfig.IsValid(); err != nil {
+	if err := defaults.Validate(ltConfig); err != nil {
 		writeResponse(w, http.StatusBadRequest, &Response{
 			Error: fmt.Sprintf("could not validate config: %s", err),
 		})
 		return
 	}
 
-	var ucConfig control.Config
+	var ucConfig interface{}
```

Comment:
Since this is not needed anymore can we remove `loadtest/control/config.go`?

Reply:
Yes, going to remove it 👍